src/storage/store.js

```diff
@@ -47,6 +47,14 @@ export const SCHEMA = {
       properties: {
         lastUsedMicDeviceId: { type: "string" }
       }
+    },
+
+    uploadPromotionTokens: {
+      type: "array",
+      items: {
+        fileId: { type: "string" },
+        promotionTokens: { type: "string" }
```

Comment:
typo here (trailing "s") -- surprised you aren't getting field validation errors -- should probably confirm this is doing what we expect